sections/Discussion.tex

```diff
@@ -1 +1,46 @@
-\section{Discussion}\label{sec:discussion}
\ No newline at end of file
+\section{Discussion}\label{sec:discussion}
+In this section, we investigate the different models to see how each meta-information field impacts the resulting topics.
+Firstly, we want to investigate the top words within each model.
+We have taken a random article, from the dataset and displayed how each model occurs within the document. 
+Before investigating the article below, we need to know a specific color scheme for each model.
+In the article, we have highlighted the highest probable words within the three most occurring topics in the article.
+The article is about agriculture and how farmers are letting their cows out onto grass fields in September. 
+It also mentions a few different farms in the Northern part of Jutland describes these in various ways.
+
+The way, we can compare these models, is by taking the top 200 words for each topic-word distribution within each model and marking them in the article below.
+Since the Author and Category models do not have a document-topic distribution, we can not look at the specific document but we have marked the given category- and author-distribution within the document, to see what the difference in topics are.
+We are looking at the top 3 topics within each model for the specific document.
+
+
+\begin{table}[h]
+	\centering
+	\caption{Color scheme for each model}
+	\begin{tabular}{l|c}
+		Topic Model & Color \\
+		\midrule
+		\Acrlong{lda} & \colorbox{red}{Red} \\
+		Author-Topic Model & \colorbox{blue}{Blue} \\

```

Comment:
denne farve er for mørk til at læse.

Reply:
jeg skal nok kigge på at finde nogle andre farver